### DISCUSSION TITLE
Instant Water

### DISCUSSION THREAD

**Discussion by matty0ung** [Brewtarget/brewtarget#732] [Q&A] [answered]
I keep meaning to ask about this.  In the miscellaneous ingredients, we list various, err, types of "Instant Water", eg "Instant Water - Munich".  What are these things?

**Comment by mikfire** [comment, marked as answer]
They were packets of dehydrated water. They fell out of fashion because nobody knew what you were supposed to add. /jk

In reality, they were packages of various salts you could add to (say) 25L of distilled water to get the water profile of Munich or Burton-On-Trent or where ever. I have no idea if these are still available or not.
